test_conformance/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ else()
     # set it, so setting it here once means we have consistent compiliation
     # flags across the entire conformance_test_* executables.
     set(CTS_EXTRA_FLAGS "${CTS_EXTRA_FLAGS} -mfpmath=sse -msse2")
-  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(ARM)|(ARM64)")
+  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(ARM)|(ARM64)|(aarch64)")
```

Comment:
I'm guessing that `ARM64` is redundant then, isn't it?

Reply:
A quick code search on GitHub find ~10k using aarch64, and ~1k using ARM64.   But both do seem to be used.

Reply:
Thanks. No harm in keeping both anyway.